### PR TITLE
Login

### PR DESCRIPTION
**Pull request by Hodaya-Hanya**
Hodaya's picture

### CODE REVIEW

File: client/api/example.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import request from 'superagent';
+import { handleSuccess, handleError } from '_utils/api';
+
```
Comment (omers4) [inline client/api/example.js]: remove

File: client/components/environment/Main/Main.js [hunk 2]
```diff
@@ -15,8 +15,8 @@ import TodoPage from '_pages/TodoPage';
 import SettingsPage from '_pages/SettingsPage';
 import LostPage from '_pages/LostPage';
 
-import Navigation from '_organisms/Navigation';
-import Footer from '_organisms/Footer';
+// import Navigation from '_organisms/Navigation';
```
Comment (omers4) [inline client/components/environment/Main/Main.js]: Remove all comments please

File: client/components/pages/HomePage/HomePage.js [hunk 3]
```diff
@@ -22,8 +22,9 @@ export default function HomePage() {
       <Section>
         <Container>
           <Title size="1">
-            Home Page
+            Home page
```
Comment (omers4) [inline client/components/pages/HomePage/HomePage.js]: why this change?

File: client/components/pages/WelcomePage/WelcomePage.js [hunk 4]
```diff
@@ -1,31 +1,46 @@
...
+  const { user } = useSelector(R.pick(["user"]));
+  const [response, setResponse] = React.useState("");
   useEffect(() => {
     if (!R.isEmpty(user)) {
-      dispatch(push('/home'));
+      dispatch(push("/home"));
     }
   }, []);
 
+  const handleAddTodo = () => {
```
Comment (omers4) [inline client/components/pages/WelcomePage/WelcomePage.js]: ?

File: package.json [hunk 5]
```diff
@@ -67,7 +63,9 @@
     "history": "^4.10.1",
     "immutability-helper": "^3.1.1",
     "immutable": "^3.8.2",
+    "jquery": "^3.6.0",
```
Comment (omers4) [inline package.json:67]: why you need this?

File: server/routes/todos.js [hunk 6]
```diff
@@ -2,10 +2,11 @@ const express = require('express');
 const { requireAuth } = require('./middleware');
 const { Todo } = require('../database/schemas');
 
-const router   = express.Router();
+const router = express.Router();
 
 module.exports = router;
 
+//DBהבאת כל המידע מהטבלה ב
```
Comment (omers4) [inline server/routes/todos.js]: No hebrew comments

**Review by omers4 — CHANGES_REQUESTED** [review]

File: client/api/example.js (hunk 1 above)
Comment (Hodaya-Hanya) [reply client/api/example.js]: Deleted example.js file

File: client/components/environment/Main/Main.js (hunk 2 above)
Comment (Hodaya-Hanya) [reply client/components/environment/Main/Main.js]: Done

File: client/components/pages/HomePage/HomePage.js (hunk 3 above)
Comment (Hodaya-Hanya) [reply client/components/pages/HomePage/HomePage.js]: It was when is play with the project in the example.. Fixed

File: client/components/pages/WelcomePage/WelcomePage.js (hunk 4 above)
Comment (Hodaya-Hanya) [reply client/components/pages/WelcomePage/WelcomePage.js]: It was when is play with the project in the example.. Don't apply this changes

File: package.json (hunk 5 above)
Comment (Hodaya-Hanya) [reply package.json:67]: I don't remember I did this changes..

File: server/routes/todos.js (hunk 6 above)
Comment (Hodaya-Hanya) [reply server/routes/todos.js]: Deleted.
